core/storage/feedback/gae_models.py

```diff
@@ -588,7 +577,9 @@ def get_message_count(cls, thread_id: str) -> int:
         return cls.get_message_counts([thread_id])[0]
 
     @classmethod
-    def get_message_counts(cls, thread_ids: List[str]) -> List[int]:
+    def get_message_counts(
+            cls, thread_ids: List[str]
+    ) -> List[Union[None, int]]:
```

Comment:
This changes the previous behaviour? I don't think we should do that here... there's no guarantee that callers handle None correctly. Is an exception/assertion more appropriate?